frontend/src/routes/Routes.tsx

```diff
@@ -6,10 +6,12 @@ import ProductCardPage from '@src/components/pages/ProductCardPage';
 import { AuthorizedOnly } from './AuthorizedOnly';
 import { EditProfilePage } from '@src/components/pages/EditProfilePage';
 import { ProductsPage } from '@src/components/pages/ProductsPage';
+import StatisticsPage from '@src/components/pages/StatisticsPage';
 
 const routes = [
   { path: '/products', element: <AuthorizedOnly><ProductsPage /></AuthorizedOnly> },
   { path: '/product/:id', element: <AuthorizedOnly><ProductCardPage /></AuthorizedOnly> },
+  { path: '/statistics', element: <AuthorizedOnly><StatisticsPage /></AuthorizedOnly> },
```

Comment:
Этот маршрут должен быть доступент только администраторам, поэтому тут нужно использовать `AdminOnly`:

```suggestion
  { path: '/statistics', element: <AdminOnly><StatisticsPage /></AdminOnly> },
```